### PR TITLE
[fix] Fix exception in api #71

### LINKED ISSUE #71: [bug] Go to can trigger exception
Found this issue while testing #70.

Subnet using this range: 10.0.0.0/16

If I try to go to 10.250.0.1, the HTTP request generates an exception: 

```
IndexError at /api/v1/subnet/7c173504-6ab6-4051-a036-9ecfe92b122f/hosts/
No exception message supplied
```

It should return a 400 bad request error which should trigger the "Invalid address" error to be shown in the go to section.

### PR DESCRIPTION
**Pull request by Purhan**
Closes #71

### CODE REVIEW

**Comment by Purhan** [comment]
@nemesisdesign I'm not sure if these are the desired changes. Please take a look whenever possible and suggest changes if required :smile:

**Review by devkapilbansal — COMMENTED** [review]
@Purhan instead of bug the type should be fix

**Comment by Purhan** [comment]
> @Purhan instead of bug the type should be fix

@devkapilbansal Could you elaborate what you mean by 'type'?

**Comment by devkapilbansal** [comment]
> > @Purhan instead of bug the type should be fix
> 
> @devkapilbansal Could you elaborate what you mean by 'type'?

In commit message you have included [bug] in title, it should be [fix]

File: openwisp_ipam/api/views.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def __len__(self):
     def index_of(self, address):
         index = int(self.subnet.subnet._address_class(address)) - self.network - 1
         if index < 0 or index >= self.count():  # pragma: no cover
-            raise IndexError
+            raise serializers.ValidationError({'detail': 'Invalid Address'})
```
Comment (nemesisdesign) [inline openwisp_ipam/api/views.py:136]: please make `Invalid Address` translatable

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
One change requested and then we can merge, thanks @Purhan

File: openwisp_ipam/api/views.py (hunk 1 above)
Comment (Purhan) [reply openwisp_ipam/api/views.py]: @nemesisdesign done

**Review by nemesisdesign — APPROVED** [review]
Thanks @Purhan :+1:

**Pull request merged**